spec/fibonacci.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+import { expect } from 'chai'
+import fibonacci from '../src/fibonacci'
+
+describe('fibonacci()', () => {
+
+it('should be a function', () => {
```

Comment:
Should add one level of indentation here and everywhere else you're writing code inside curly braces. It makes for more readable code and shows good consistency.